pkg/registry/pod/etcd/etcd_test.go

```diff
@@ -846,6 +847,33 @@ func TestEtcdCreate(t *testing.T) {
 	}
 }
 
+func TestEtcdCreateBindingNoPod(t *testing.T) {
```

Comment:
Please add a comment describing what this is trying to test (very brief summary of the scenario you are creating). Something like "Ensure that when scheduler creates a binding for a pod that has already been deleted by the API server, API server returns not-found error."